tests/test_normalize_fields.py

```diff
@@ -0,0 +1,109 @@
+from concurrent.futures import ThreadPoolExecutor
+import unittest
+from datetime import datetime, timezone
+
+import preql
+
+from data_diff.database import BigQuery, MySQL, Snowflake, connect_to_uri, Oracle, DEFAULT_PRECISION
+from data_diff.sql import Select
+from data_diff import database as db
+
+from .common import CONN_STRINGS
+
+
+DATE_TYPES = {
+    db.Postgres: ["timestamp({p}) with time zone", "timestamp({p}) without time zone"],
```

Comment:
How exactly do the time zone and without timezone compare correctly? I guess it works because you always drop the timezone bit?

Should we include the timezone in the string comparison if _both_ types have a timezone, to ensure they're the same? I don't think we have to do that now, and I know it's a bit tricky for e.g. MySQL where we have to figure out what timezone MySQL is in to get it (but it's doable).

I feel it's more important that we have support for floats and have proper types for all "type categories" we support more so than this though.

Reply:
> Should we include the timezone in the string comparison if both types have a timezone, to ensure they're the same?

I don't know. But I think it's not a good idea.